server/lib/api/v1/exceptions.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  module V1
+    module Exceptions
+      class Error < StandardError
+        attr_accessor :code, :status, :title, :detail, :href
+
+        def initialize(code:, status:, title:, detail:, href:)
+          @code = code
+          @status = status
+          @title = title
+          @detail = detail
+          @href = href
+        end
+      end
+
+      class WrongContentTypeError < Error
+        def initialize
+          super code: 1001, status: 415, title: "Unsupported Media Type", detail: "Content Type must be application/vnd.api+json", href: "http://shaojunda.coding.me/doc/api_doc.html"
```

Comment:
Can you remove the personal site href ?

Reply:
Please remove this url or replace it with a better url.

Reply:
ok, temporarily use a fake doc link, I will replace it when all is done